Assets/MixedRealityToolkit/InputSystem/Cursors/BaseCursor.cs

```diff
@@ -3,6 +3,7 @@
 
 using Microsoft.MixedReality.Toolkit.Internal.Definitions.InputSystem;
 using Microsoft.MixedReality.Toolkit.Internal.Definitions.Physics;
+using Microsoft.MixedReality.Toolkit.Internal.Devices.WindowsMixedReality;
```

Comment:
WOAH, why is there a WindowsMixedReality reference in Cursors.  
Only the devices space and the MR manager should have ANY knowledge of the Devices

Reply:
When we ask the controller if it's a hand below

Reply:
Then we need a way for the framework to realise it.  NOT pull it from the device platform.

Reply:
Ok, we'll create a task to review these requirements later